tools/build/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Fri Nov 08 10:58:59 PST 2019
+distributionBase=GRADLE_USER_HOME

```

Comment:
nit: line endings?

Reply:
This script is ultimately owned by Gradle. I didn't author it. It's part of Gradle's tooling.